ckanext/versioning/logic/helpers.py

```diff
@@ -118,3 +116,16 @@ def get_dataset_current_revision(dataset_name):
     backend = get_metastore_backend()
 
     return backend.fetch(dataset_name).revision
+
+def get_dataset_version(package_id, revision_ref):
+    '''Get the DatasetVersion for a package and revision_ref.
+    '''
+    version = model.Session.query(DatasetVersion). \
```

Comment:
Is this still here b/c we have not yet refactored to get revision tags from MetaStore?